### ISSUE TITLE
Firmware source code

### ISSUE THREAD

**Issue by cquiroz** [ai03-2725/Soyuz#1]
First, thanks for the keyboard. It looks great and i think it will be very useful for me.

Do you have the source code for the firmware. I want to customize my Soyuz layout

**Comment by cquiroz** [comment]
I’m guessing the firmware is qmk based but maybe it something else?

**Comment by nil0ka** [comment]
Hello, @cquiroz san.

I just customized Soyuz layout. and I share the way.
I used [Keyboard Firmware Builder](https://kbfirmware.com/)

1. upload [Basic-Firmware/soyuz-kbfirmware.com-template.json](https://github.com/ai03-2725/Soyuz/blob/master/Basic-Firmware/soyuz-kbfirmware.com-template.json).
2. choose 'KEYMAP' and select the key, layer to customize you want. and configure a keymap.
3. when configure all key are complete, compile to '.hex' file and download to your local.
4. finally, you write to Microcontroller created '.hex' file use 'QMK Toolbox'.

I successed to customize Soyuz this way.
I hope that it is useful to you :)
